addons/account/models/chart_template.py

```diff
@@ -466,31 +476,41 @@ def deref(values, model):
 
         def defer(all_data):
             """Defer writing some relations if the related records don't exist yet."""
+
+            def should_defer(created_models, yet_to_be_created_models, model, field_name, field_val):
+                field = self.env[model]._fields.get(field_name, None)
+                if not (
+                    field
+                    and field.relational
+                    and field.comodel_name not in created_models
+                    and field.comodel_name in [model] + yet_to_be_created_models):
+                    return False
+                elif isinstance(field_val, list | tuple):
+                    # Check recursively if there are subfields that should be deferred
+                    sub_defer = False
```

Comment:
Useless variable, just return instead of breaking the control loop (and not breaking it fully/correctly)